core/src/main/resources/hudson/diagnosis/NullIdDescriptorMonitor/message_fr.properties

```
@@ -0,0 +1,3 @@
+blurb=Les extensions suivantes n''ont aucune valeur "ID" et peuvent en conséquence causer des problèmes. Veuillez mettre à jour ces plugins s''ils ne sont pas dans la dernière version, \
+  et s''ils sont déjà dans la dernière version, merci de remonter le problème afin que nous puissions le corriger
```

```suggestion
  et s''ils sont déjà dans la dernière version merci de remonter le problème afin que nous puissions le corriger
```
Comma not mandatory in French here, increase the link between these two parts (subjective)